scripts/install_static_z3.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+git clone --depth 1 --branch z3-4.8.1 https://github.com/Z3Prover/z3.git
+cd z3
+mkdir build
+cd build
+LDFLAGS="-static" cmake -DBUILD_LIBZ3_SHARED=OFF ..
```

Comment:
If we don't want to keep the script, I could instead just make a note to myself that this is how to do it...

Reply:
Removed it and created an issue with a note instead.